install/operator/pkg/syndesis/olm/csv_test.go

```diff
@@ -32,9 +33,14 @@ func Test_csv_build(t *testing.T) {
 	assert.NoError(t, err)
 
 	c := &csv{config: conf, operator: "operator-image"}
+
+	pkgPath := "pkg"
+
+	_ = os.Mkdir(pkgPath, os.ModeDir)
+	defer os.Remove(pkgPath)
```

Comment:
Sad, this bit, seems we took on some debt by necessitating the `pkg` directory.

Reply:
gone.